lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -1939,3 +1943,35 @@ static void _generate_dl_flow(
     }
   }
 }
+
+static bool does_bearer_context_hold_valid_enb_ip(
+    ip_address_t enb_ip_address_S1u) {
+  OAILOG_FUNC_IN(LOG_SPGW_APP);
+  struct in6_addr ipv6_address = {0};
```

Comment:
can you define this as static to avoid initializing address on every function call.